tests/virtualization/universal/download_image.pm

```diff
@@ -0,0 +1,36 @@
+#Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Download disk image
+# Maintainer: Pavel Dostal <pdostal@suse.cz>
+
+use base "consoletest";
+use virt_autotest::common;
+use virt_autotest::utils;
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    my $self = shift;
+
+    ensure_default_net_is_active();
+
+    assert_script_run "if [ ! -d \"/mnt/pandora\" ]; then mkdir /mnt/pandora; fi";
+    assert_script_run "mount pandora.suse.cz:/var/lib/libvirt/images /mnt/pandora";
+
+    zypper_call("install rsync") if (script_run("which rsync") != 0);
+    #assert_script_run "if [ -f \"/var/lib/libvirt/images/win2k19.raw\" ]; then rm /var/lib/libvirt/images/win2k19.raw; fi";
+
+    assert_script_run "if [ ! -f \"$virt_autotest::common::imports{$_}->{disk}\" ]; then rsync -v --progress $virt_autotest::common::imports{$_}->{source} $virt_autotest::common::imports{$_}->{disk}; fi", 600 foreach (keys %virt_autotest::common::imports);
```

Comment:
I would add a note for better understanding here. Like

    # Pull images from server if necessary